src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Proposal/Proposal.ts

```diff
@@ -235,7 +235,8 @@ export var detailDirective = (
             path: "@",
             processOptions: "="
         },
-        link: (scope : IScope) => {
+        link: (scope) => {
```

Comment:
Good question. I probably briefly put a new variable into the scope, but I don't remember.